netlib/tcp.py

```diff
@@ -901,7 +901,7 @@ def handle_error(self, connection_, client_address, fp=sys.stderr):
         """
         # If a thread has persisted after interpreter exit, the module might be
         # none.
-        if traceback:
+        if traceback and six:
```

Comment:
That code can execute after modules have been unloaded (interpreter exit with a persistent thread). If the six module or the traceback module is gone, there's nothing we can do, so we ignore rather than crashing.